src/chrome/content/code/HTTPSRules.js

```diff
@@ -622,17 +719,15 @@ const HTTPSRules = {
    * This function is only defined for fully-qualified hostnames. Wildcards and
    * cookie-style domain attributes with a leading dot are not permitted.
    * @param host {string}
-   * @return {Array.<RuleSet>}
+   * @return true iff we didn't have to go async to load rules
    */
-  potentiallyApplicableRulesets: function(host) {
+  potentiallyApplicableRulesets: function(host, callback) {
+    if (!callback) {
```

Comment:
It looks like there aren't any call sites that could cause this branch to execute, but OK...